src/plots/plots.js

```diff
@@ -1104,8 +1104,16 @@ plots.purge = function(gd) {
     // remove modebar
     if(fullLayout._modeBar) fullLayout._modeBar.destroy();
 
-    if(gd._transitionData && gd._transitionData._animationRaf) {
-        window.cancelAnimationFrame(gd._transitionData._animationRaf);
+    if(gd._transitionData) {
+        // Ensure any dangling callbacks are simply dropped if the plot is purged.
+        // This is more or less only actually important for testing.
```

Comment:
Aiming for _useful_ comments. 😄 

![coding](https://cloud.githubusercontent.com/assets/572717/20160330/274f0e92-a6b4-11e6-93b0-db9b81cc1213.png)